src/cms/templates/feedback/admin_feedback_list.html

```diff
@@ -0,0 +1,121 @@
+{% extends "_base.html" %}
+{% load i18n %}
+{% block content %}
+{% load static %}
+{% load widget_tweaks %}
+<h2>Technical Feedback</h2>
+
+<table class="w-full mt-4 rounded border border-solid border-gray-200 shadow bg-white">
+    <tr class="border-b border-solid border-gray-200">
+        <th class="text-sm text-left uppercase py-3 px-2 min" >{% trans 'Type' %}</th>
+        <th class="text-sm text-left uppercase py-3 px-2 min" >{% trans 'Object' %}</th>
+        <th class="text-sm text-left uppercase py-3 px-2 min" >{% trans 'Comment' %}</th>
+        <th class="text-sm text-left uppercase py-3 px-2 min" >{% trans 'Rating' %}</th>
+        <th class="text-sm text-left uppercase py-3 px-2 min" >{% trans 'Date' %}</th>
+    </tr>
+    {% if not feedback_Admin_exists %} 
+    <tr><td colspan="5" class="px-2 py-3">{% trans 'Sorry No Feedback available yet!' %} </td></tr> 
+    
+    {% else %}
+    {% for feedback in page_feedback %}
+     <tr>
+        <td align="center">{% trans 'Page' %}</td>
+        <td align="center">
+            <a href="{% url 'edit_page' page_id=feedback.page_translation.page.id region_slug=feedback.page_translation.page.region.slug language_code=feedback.page_translation.language.code %}">
+                {{ feedback.page_translation.title }}
+            </a>
+        </td>
+        <td align="center">{{ feedback.comment }}</td>
+        <td align="center">{{ feedback.get_emotion_display }}</td>
+        <td align="center">{{ feedback.created_date }}</td>
```

Comment:
The table head is fine now, please repeat the process for all table fields, not only the `<th>`.
As an example, see the page feedback suggestion below. Please adapt this to all table fields, also in the region feedback template.
```suggestion
     <tr class="border-t border-solid border-gray-200 hover:bg-gray-100">
        <td class="pl-2">{% trans 'Page' %}</td>
        <td>
            <a href="{% url 'edit_page' page_id=feedback.page_translation.page.id region_slug=feedback.page_translation.page.region.slug language_code=feedback.page_translation.language.code %}"  class="block py-3 px-2 text-gray-800">
                {{ feedback.page_translation.title }}
            </a>
        </td>
        <td>{{ feedback.comment }}</td>
        <td>{{ feedback.get_emotion_display }}</td>
        <td class="pr-2">{{ feedback.created_date }}</td>
```